docs/source/modeling_with_aequilibrae/route_choice/route_choice_model.rst

```diff
@@ -0,0 +1,124 @@
+Route choice models
+===================
+
+As argued in the literature [3]_, the route choice problem does not have a closed solution, and the selection
+of one of the many modelling frameworks [4]_ depends on many factors. A common modelling framework in practice
+is explicit path generation, where the route choice problem can be broken down into two sub-problems: Choice set
+generation and the choice selection process. Most choice set generation algorithms are versions of the k-shortest path
+algorithm.
+
+Path-Size Logit (PSL)
+~~~~~~~~~~~~~~~~~~~~~
+
+Path-Size logit is based on the multinomial logit (MNL) model, which is one of the most used models in the
+transportation field in general (Ben-Akiva and Lerman, 1985). It can be derived from random utility-maximizing
+principles with certain assumptions on the distribution of the random part of the utility. To
+account for the correlation of alternatives, Ramming (Ramming, 2002) introduced a correction factor that measures the
+overlap of each route with all other routes in a choice set based on shared link attributes, which gives rise to the PSL
+model. The PSL model’s utility function is defined by
+
+.. math:: U_i = V_i + \beta_{PSL} \cross \log{\gamma_i} + \varepsilon_i
+
+with path overlap correction factor
+
+.. math:: \gamma_i = \sum_{a \in A_i} \frac{l_a}{L_i} \cross \frac{1}{\sum_{k \in R} \delta_{a,k}}
```

Comment:
our implementation does not use length, but (negative) utility, right?

Reply:
I think this is from our ATRF paper, for which I calculated the path overlap factors with length "by hand" (in my estimation notebooks). I checked the influence on estimation results and it didn't matter, but we should probably make sure the documentation is consistent with our implementation.